staging/src/k8s.io/client-go/tools/leaderelection/leaderelection_test.go

```diff
@@ -48,15 +48,18 @@ func createLockObject(objectType, namespace, name string, record rl.LeaderElecti
 		objectMeta.Annotations = map[string]string{
 			rl.LeaderElectionRecordAnnotationKey: string(recordBytes),
 		}
+		objectMeta.Labels = map[string]string{}
```

Comment:
You could initialize the Labels map outside the switch statement

Reply:
done